packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal-header.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { DemoLight, DemoDark } from '../../../public/images/index';
+import classNames from 'classnames';
+import { Badge, GradientBackground, Icon, Text, Watermark } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { getWalletCurrencyIcon } from 'Constants/utils';
+import { getCashierWalletModalBackgrounds } from 'Constants/wallet-backgrounds';
+
+type TCashierWalletModalHeaderProps = {
+    balance?: string | number;
+    currency: string;
+    is_dark: boolean;
+    is_demo: boolean;
+    is_mobile: boolean;
+    landing_company_shortcode: 'svg' | 'malta';
+    closeModal: VoidFunction;
+    show_wallet_name: boolean;
+};
+
+const CashierWalletModalHeader = ({
+    balance,
+    closeModal,
+    currency,
+    is_dark,
+    is_demo,
+    is_mobile,
+    landing_company_shortcode: shortcode,
+    show_wallet_name,
+}: TCashierWalletModalHeaderProps) => {
+    const header_class_name = 'cashier-wallet-modal__header';
+
+    const getBadgeLabel = React.useCallback(() => {
+        if (is_demo) return 'Demo';
+        if (['svg', 'malta'].includes(shortcode)) return shortcode.toUpperCase();
+        return '';
+    }, [is_demo, shortcode]);
+
+    const getCloseIcon = React.useCallback(() => {
+        if (is_demo && is_dark) return 'IcAppstoreCloseLight';
+        if (is_demo && !is_dark) return 'IcAppstoreCloseDark';
```

Comment:
```suggestion
        if (is_demo) return 'IcAppstoreCloseDark';
```
You can ignore it too